CONTRIBUTING.md

```diff
@@ -95,11 +95,53 @@ Remember that:
 
 For more information on forks and pull request, check out [this GitHub guide](https://guides.github.com/activities/forking/).
 
-### SOFA Contributor License Agreement ("Agreement"), v1.0
 
-Note that by submitting a "Contribution" (shall mean the code, documentation or other original works of authorship) to SOFA, through the Pull Request mechanism in github or by other means, "You" (the entity that is making this Contribution) hereby accept and agree to the following terms and conditions for Your present and future Contributions:
+### SOFA Developer Certificate Of Origin (DCO)
 
-- You certify that You have authored 100% of the content, or in the event part or all of the content has been authored by a third party, you have appropriate permissions to include their work in your Contribution,
-- You certify You have the necessary rights to license the Contribution (i.e. if You are employed You have received the necessary permissions from Your employer to make the Contribution),
-- You grant a perpetual, irrevocable, non-exclusive, royalty-free, worldwide license to distribute this Contribution under the license agreement commonly known as the [LGPL License](https://opensource.org/licenses/LGPL-2.1).
-- You will remain the copyright holder of your Contribution.
+If you are contributing code or documentation to the SOFA project, and using the git signed-off-by mechanism, you are agreeing to this certificate.  This DCO essentially means that:
```

Comment:
Maybe a small introduction: 
_SOFA is using the [mechanism of the linux project](https://www.kernel.org/doc/html/latest/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin) to track and secure all issues related to copyrights: the Developper Certificate of Origin (DCO) ._